apps/web/src/pages/Profile/AccountSettings/NotificationSettings.tsx

```diff
@@ -0,0 +1,262 @@
+  ignoredInAppNotifications: NotificationFamily[];
+}
+
+const UpdateIgnoredNotifications_Mutation = graphql(/* GraphQL */ `
+  mutation updateIgnoredNotifications(
+    $ignoredEmailNotifications: [NotificationFamily]
+    $ignoredInAppNotifications: [NotificationFamily]
+  ) {
+    updateIgnoredNotifications(
+      ignoredEmailNotifications: $ignoredEmailNotifications
+      ignoredInAppNotifications: $ignoredInAppNotifications
+    ) {
+      id
+      ignoredEmailNotifications
+      ignoredInAppNotifications
+    }
+  }
+`);
+
+const NotificationSettings = ({
+  ignoredEmailNotifications,
+  ignoredInAppNotifications,
+}: NotificationSettingsProps) => {
+  const intl = useIntl();
+  const [, executeMutation] = useMutation(UpdateIgnoredNotifications_Mutation);
+
+  const methods = useForm<FormValues>({
+    defaultValues: dataValuesToFormValues({
+      ignoredEmailNotifications,
+      ignoredInAppNotifications,
+    }),
+  });
+
+  const notificationOptions = (id: string): CheckboxOption[] => [
+    {
+      value: "email",
+      label: (
+        <span data-h2-visually-hidden="base(invisible)">
+          {intl.formatMessage(commonMessages.email)}
+        </span>
+      ),
+      disabled: id === "systemMessages",
+    },
+    {
+      value: "inApp",
+      label: (
+        <span data-h2-visually-hidden="base(invisible)">
+          {intl.formatMessage(commonMessages.inApp)}
+        </span>
+      ),
+      disabled: id === "systemMessages",
+    },
+  ];
+
+  const { handleSubmit } = methods;
+
+  const handleError = () => {
+    toast.error(
+      intl.formatMessage({
+        defaultMessage: "Error: could not update applicant settings",
+        id: "J1YiBm",
+        description:
+          "Message displayed when an error occurs while updating applicant settings.",
+      }),
+    );
+  };
+
+  const handleFormSubmit: SubmitHandler<FormValues> = async (
+    values: FormValues,
+  ) => {
+    const data = formValuesToData(values);
+    await executeMutation(data)
+      .then((result) => {
+        if (result.data?.updateIgnoredNotifications) {
+          toast.success(
+            intl.formatMessage({
+              defaultMessage: "Updated applicant settings",
+              id: "zV8YQ1",
+              description:
+                "Message displayed when applicant updated their settings.",
+            }),
+          );
+        } else {
+          handleError();
+        }
+      })
+      .catch(() => {
+        handleError();
+      });
+  };
+
+  const formFields = [
+    {
+      id: "systemMessages",
+      name: "systemMessages",
+      legend: intl.formatMessage({
+        defaultMessage: "System Messages",
+        id: "KzpzIE",
+        description: "Title for system messages checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "These notifications are required and include email verification and other important communications.",
```

Comment:
Why is there a checkbox to control them that is not able to be unchecked? Why have the System messages section at all?

Reply:
That's a design question, but I'm guessing just to show the user that it can't be changed 🤷

Reply:
Yes - this is to set the expectation that by having an account, there are certain emails/notifications you will always received. It's misleading to allow a user to "turn them all off" and then still send them notifications. This is an easy way of showing them that there is a category of notifications they will always get.

Reply:
@substrae Eric's advice is that disabled checkboxes is only appropriate in a very narrow circumstance where the input is temporarily disabled while other stuff on the page happens. I did a quick search and people on the internet say that the checked/unchecked status of the checkbox is no longer informative if the checkbox is disabled.

Do you have any thoughts on other ways we could implement this?

Quick ideas:
- Replace checkboxes with words "always on" or similar.
- Use icons to represent the disabled checkboxes

Reply:
I explained in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971#discussion_r1562519928